tests/RepositoryForms/Form/DataTransformer/TranslatablePropertyTransformerTest.php

```diff
@@ -62,8 +62,8 @@ public function testReverseTransform($inputValue, $languageCode, $expected)
     public function reverseTransformProvider()
     {
         return [
-            [false, 'fre-FR', null],
-            [null, 'fre-FR', null],
+            [false, 'fre-FR', ['fre-FR' => false]],
```

Comment:
not so sure about storing false for false. I would stick to `null` when empty.

Reply:
@alongosz basically, I do not think there is a point at even testing for "false", there won't be any situation that would output us a "false" there. When it's empty - it's just "null".

Reply:
Testing for false is good, as those are unit tests and we should test for unexepected values are there is no typehinting there.

I would just say, it should be `[false, 'fre-FR', ['fre-FR' => null]],`

@alongosz

Reply:
@ViniTou resolved in https://github.com/ezsystems/repository-forms/pull/338/commits/847c0b6268e4ec76df0b991892ea39b6af3b9569